Modules/_testcapimodule.c

```diff
@@ -2587,6 +2587,117 @@ get_datetime_fromtimestamp(PyObject* self, PyObject *args)
     return rv;
 }
 
+static PyObject *
+get_fields_from_date(PyObject* self, PyObject *args)
+{
+    PyObject *obj = NULL;
+    PyObject *rv;
+    int year, month, day;
+
+    if (!PyArg_ParseTuple(args, "O", &obj)) {
+        return NULL;
+    }
+    
```

Comment:
It's not showing up in the GH interface, but on my local checkout this line has trailing whitespace. Can you check and see if that's the case for you, and if so remove it?

Reply:
I saw a trailing space. I tried to remove it. Please confirm.